### PR TITLE
Fix send email

### PR DESCRIPTION
**Pull request by djino97**

### CODE REVIEW

**Review by BelehovEgor — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/UserService.Business/Commands/User/FindUserCommand.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ private IFindDepartmentUsersResponse GetUserIdsByDepartment(Guid departmentId, i
             try
             {
                 var request = IFindDepartmentUsersRequest.CreateObj(departmentId, skipCount, takeCount);
-                var response = _requestClient.GetResponse<IOperationResult<IFindDepartmentUsersResponse>>(request, timeout: TimeSpan.FromSeconds(2)).Result;
+                var response = _requestClient.GetResponse<IOperationResult<IFindDepartmentUsersResponse>>(request, timeout: RequestTimeout.Default).Result;
```
Comment (DarkFelas) [inline src/UserService.Business/Commands/User/FindUserCommand.cs:32]: use new kernel version and remove RequestTimeout.Default

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 2]
```diff
@@ -111,33 +109,18 @@ private void SendEmail(DbUser dbUser, string password, List<string> errors)
...
-                }
-
-                emailRequest = ISendEmailRequest.CreateObj(
-                        rcGetTemplateTagsResponse.Body.TemplateId,
-                        senderId,
-                        email.Value,
-                        templateLanguage,
-                        templateValues);
                 IOperationResult<bool> rcSendEmailResponse = _rcSendEmail
                     .GetResponse<IOperationResult<bool>>(emailRequest, timeout: RequestTimeout.Default)
```
Comment (DarkFelas) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:125]: remove RequestTimeout

File: src/UserService.Business/Commands/Password/ForgotPasswordCommand.cs [hunk 3]
```diff
@@ -63,33 +61,17 @@ private Guid SetGuidInCache(Guid userId)
...
-                IOperationResult<bool> response = _rcSendEmail.GetResponse<IOperationResult<bool>>(
-                    ISendEmailRequest.CreateObj(
-                        rcGetTemplateTagsResponse.Body.TemplateId,
-                        senderId,
-                        email,
-                        templateLanguage,
-                        templateValues
-                       )).Result.Message;
+                IOperationResult<bool> response = _rcSendEmail
+                    .GetResponse<IOperationResult<bool>>(emailRequest, timeout: RequestTimeout.Default)
```
Comment (DarkFelas) [inline src/UserService.Business/Commands/Password/ForgotPasswordCommand.cs:72]: remove timeout

File: Common.Configurations.targets [hunk 4]
```diff
@@ -34,7 +34,7 @@
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
     <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.27" />
```
Comment (DarkFelas) [inline Common.Configurations.targets:36]: use new kernel

**Pull request merged**